<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+] Status</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Sign in</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="usrLabel">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pwdLabel">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="usrEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="pwdEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <zorder></zorder>
   <zorder></zorder>
   <zorder>LoginButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>usrEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>LoginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwdEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>LoginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
